### ISSUE TITLE
[Bug]:  Apply EntityGraph Definition on Query Method

### ISSUE THREAD

**Issue by hantsy** [jakartaee/data#752]
### Specification Version

1.0.0-RC1

### Bug report

If possible to apply the Entity Graph definitions on Entities to a specific query method in the Repositories? Spring Data JPA provides similar features for developers.

### Additional information

For example,

```java
@Entity
@NamedEntityGraph(name="with-comments")
class Post{
   @OneToMany
   Set<Comment> comments
}

````

```java
interface PostRepository {

    @Find
    @EntityGraph(name="with-comments")
    Post findById(Long id);
}

```

**Comment by gavinking** [comment]
This is not in the spec, since it would be a very Java Persistence-specific feature. 

Also, the annotations for defining named entity graphs are super-ugly and I'm not a fan of them at all.

We do already have a _very_ similar feature to this in Hibernate Data Repositories, but it uses Hibernate's named fetch profiles stuff, which have a much more elegant annotation-based definition.

**Comment by gavinking** [comment]
(When we define integration with Jakarta Persistence, this is something we can look at.)

**Comment by hantsy** [comment]
Yes, it should be part of the JPA integration, but it seems there is no a JPA extended spec `jakarta-data-jpa` under Jakarta Data.

**Comment by njr-11** [comment]
Jakarta Data is only in its 1.0 version.  It is not a bug that Jakarta Data does not offer the suggested function.  This issue should be an enhancement request, not a bug.
